pkgs/development/compilers/djgpp/default.nix

```diff
@@ -0,0 +1,137 @@
+{ bison, buildPackages, curl, file, fetchFromGitHub, fetchurl, flex, i686Build ? false
+, lib, makeWrapper, perl, stdenv, texinfo, unzip, which }:
+
+stdenv.mkDerivation rec {
+  pname = "djgpp";
+  version = "12.1.0";
+  buildVersion = if i686Build then "${version}-i686" else "${version}";
+
+  # adapted from https://github.com/andrewwutw/build-djgpp/blob/master/script/12.1.0
```

Comment:
```suggestion
  # adapted from https://github.com/andrewwutw/build-djgpp/blob/master/script/12.1.0
  env = {
```

Reply:
@AndersonTorres actually I'm not sure about this, the values aren't used in any of the subsequent shell contexts, is it enough just to group them? I don't know how fetchUrl can access an envvar.

Reply:
Ah, they are not "shell vars"? You are using them merely as `#define`s (to borrow a C preprocessor keyword)?

This is a bit more complicated. For now, forget the `env` then.